orchagent/aclorch.cpp

```diff
@@ -5112,3 +5169,48 @@ void AclOrch::removeAllAclRuleStatus()
     }
 }
 
+// Bind egress ACL table (with bind type switch) to switch
+bool AclOrch::bindEgrAclTableToSwitch(AclTable &table)
```

Comment:
Will be good to move this binding logic and invoking switch SAI API to switchOrchagent.

Reply:
Since there are other functions in the same file which were using switch API to get information, so it was added here. I believe it should be ok.

Reply:
Please move the ACLTable binding to switch APIs to switchorch. We already have AclGroup switch bind in switchOrch.

Reply:
Hi @kperumalbfn, moved `bindEgrAclTableToSwitch` and `unbindEgrAclTableFromSwitch` to switchorch.